docs/tutorials/06-batch-infer.ipynb

```diff
@@ -695,10 +690,68 @@
    ],
    "source": [
     "project.enable_model_monitoring(\n",
-    "    base_period=1, wait_for_deployment=True, deploy_histogram_data_drift_app=True\n",
+    "    base_period=1, wait_for_deployment=True, deploy_histogram_data_drift_app=False\n",
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "63d483c2",
+   "metadata": {},
+   "source": [
+    "## Change the *histogram data drift* application defaults \n",
```

Comment:
Italic are not really used in the doc. To keep the look n feel - suggest either removing them, or using histogram_data_drift